### DISCUSSION TITLE
The Fraction absorbed value was exceed 1 when it was simulated using EHC parameters.

### DISCUSSION THREAD

**Discussion by hjhj112233** [Open-Systems-Pharmacology/Forum#934] [Questions & Problems] [answered]
Dear community, 

I have PK observed data which has double peak. So I optimized many parameters such as EHC, gallbladder, transporters, biliary clearance and meal...… and then I made double peak simulation. 

Simulated PK parameters (Cmax, AUC…) fitted well with it’s observed data. But I wonder why Fraction absorbed was over 1. My Fraction absorbed value was 2.11 and I think it is impossible to make sense. The PK-sim manual said that "Absorbed fraction of applied oral dose. Please note that, e.g. in the case of enterohepatic circulation, **this fraction may exceed 1**". 

Is it just limitation of PK-sim?
Please share your opinions.... Thanks

![DP](https://user-images.githubusercontent.com/87287996/125231044-fe494d80-e314-11eb-8eca-a03001c51c13.jpg)

**Comment by tobiasK2001** [comment, marked as answer]
Hi, Sorry for answering so late.
But I don't think your fraction absorbed of 2.11 is a limitation of PK-Sim. It just tells you that your compound molecules are absorbed more than once due to your EHC assumption. And it is quite physiological to assume that in case of enterohepatic **cycling** you have a reuptake of excreted compound. It might prompt you to think about your assumption. I am not sure if EHC has been found for Glipizide? Is your compound a substrate for bilary transporters?
You might think of other possible explanation for double peaks. E.g. check this out https://doi.org/10.1208/s12248-020-00548-8 and look at figure 3. Nefadozone-HCl. Pepin et al could rule out EHC and discuss two other possibilities coming from the absorption side:
A) Since both nefazodone-HCl solubility and permeability are high, this drug could be a marker of gastric emptying and partial gastric emptying has been reported for other products in the literature.
(B) Wettability issues and consequently slow dissolution rates are reported for nefazodone-HCl tablets, despite the high measured equilibrium solubility.

Hope that might help you.

Best, Tobias
